app/js/helpers/apiHelper.js

```diff
@@ -69,3 +69,5 @@ export class ApiHelper {
     return this.build(requestUrl, 'DELETE').send();
   }
 }
+
+export const BASE_URL = "https://addons.openmrs.org/api/v1/addon";
```

Comment:
Refactor looks fine. Did you consider putting the `BASE_URL` as a static member in the `ApiHelper` class? That way, there's only one import variable.

Reply:
Thank you for the observation, i have also updated the PR